kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -170,6 +169,7 @@
     },
     created() {
       window.addEventListener('click', this.handleWindowClick);
+      this.$kolibriTheme = global.kolibriTheme;
```

Comment:
Why are we directly referencing the global here?

Reply:
The top-level `global.kolibriTheme` contains kolibri-specific config, e.g. logo behaviors adn styling. The generic stuff is used here: https://github.com/learningequality/kolibri/pull/5847/files#diff-e46818b93613a96345b99fe7bbe04beaR15

Reply:
I would rather that we don't rely on the global, and have everything go through the theme mixin, so that we have a single point for relying on the globals (similarly to how we only use the language globals in the `i18n` functionality and then everything else relies on imports from that module).

Reply:
okay - I can make an independent mixin for the config options that are kolibri-specific

Reply:
to be addressed using new `pluginData` functionality

Reply:
Noting that to do this, it would still have to be defined in a core module somewhere, in case it was to be referenced outside of the `default_frontend` bundle.